src/ngx_http_lua_module.c

```diff
@@ -384,6 +384,13 @@ static ngx_command_t ngx_http_lua_cmds[] = {
       offsetof(ngx_http_lua_loc_conf_t, use_default_type),
       NULL },
 
+      { ngx_string("lua_semaphore_threshold"),
+      NGX_HTTP_MAIN_CONF|NGX_CONF_TAKE1,
```

Comment:
Style: please be careful with indentation. See other entries for examples. Thank you :)